### PR TITLE
Improve test build

### PR DESCRIPTION
**Pull request by drwells**
Migrated from #1289. Depends on #1289 (we need verbose mode).

@vibraphone These changes are needed to get the test suite running - I also disabled all the very expensive tests since the virtual machine is pretty slow.

### CODE REVIEW

File: .github/test-build [hunk 1]
```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
+set -eou pipefail
 
 # Remember where we started; this is where our source checkout lives.
-src=`pwd`
+src=$(pwd)
 
 # Load the module command and use it to load MPI.
-. /etc/profile.d/modules.sh
+./etc/profile.d/modules.sh
```
Comment (drwells) [inline .github/test-build:8]: I'm about 99% sure this was a typo but we should verify that.

Comment (vibraphone) [inline .github/test-build:8]: This is not correct? `. script` is equivalent to `source script`. Is there even a `./etc` directory that exists?

Comment (drwells) [reply .github/test-build:8]: You're right - let me fix this again.

Comment (drwells) [reply .github/test-build:8]: I got this wrong (see below).

**Comment by drwells** [comment]
I think this has been superceded by https://github.com/vibraphone/IBAMR/pull/2 so it doesn't need to be reviewed.

**Review by boyceg — APPROVED** [review]

**Pull request closed without merging**